tests/importer/data/safari_audiobook.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@ntarocco, @zzacharo  I had some second thoughts about exposing the below metadata in the tests, but since the e-book is behind login I think it is fine, WDYT?

Reply:
what exactly are you worried about in terms of metadata exposure?

Reply:
@zzacharo if we can use the book as an example in tests

Reply:
I don't see any private data in the test record, but worst case you can alter some of the metadata.